README.rst

```diff
@@ -75,6 +75,12 @@ to see if it will succeed.
 Build Documentation Locally
 ---------------------------
 
+To install dependencies for building the documentation::
+
+    pip install Sphinx
+    pip install sphinx_rtd_theme
+    pip install numpydoc
+
```

Comment:
since we have specific version requirements, it might be easier to suggest `pip install dev-requirements.txt` here.

Reply:
I thought about that, but `dev-requirements.txt` would also install dependencies for running tests, which this section of the README isn't about.

Reply:
Or maybe we should have a "Local Development" instead? That way, we can point to `dev-requirements.txt` and explain how to run tests and build the doc locally.

Reply:
I thinking adding a section on local development is out of scope for this.  I can see how the irrelevant stuff in `dev-requirements.txt` might make it hard to install so we can leave this as is.